doc/modules/neighbors.rst

```diff
@@ -711,10 +717,10 @@ added space complexity in the operation.
 
 .. topic:: References:
 
```

Comment:
I think that these need either to be in a "topic", or in as footnotes: currently, they do not render right
https://48180-843222-gh.circle-artifacts.com/0/doc/modules/neighbors.html#transform

This is because the indentation is not correct.

You could remove the "topic" block, and add the following:

```
___________

**References**

```

Where the '__________' inserts an hrule.

Reply:
Thanks, I went for fixing the indentation, it should work like at the end of this section: https://github.com/scikit-learn/scikit-learn/blob/master/doc/modules/decomposition.rst#truncated-singular-value-decomposition-and-latent-semantic-analysis
I'll try to build the doc locally to be faster than circleci to check if it works

Reply:
I just saw it and it works :)